README.md

```diff
@@ -134,6 +134,30 @@ Datadog.configure { |c| c.ci.agentless_mode_enabled = true }
 
 ## Additional configuration
 
+### Experimental test suite level visibility support
+
+Test suite level visibility (TSLV) is available for `datadog-ci`` gem versions 0.6 and above.
```

Comment:
Minor: I find it somewhat weird to list the minimum version for a feature in the readme, since, well, every feature that gets added is a feature that has a minimum version.

Reply:
this is very true, I think this info is not needed here